src/Auth0Client.ts

```diff
@@ -817,8 +817,8 @@ export class Auth0Client {
    * remaining before expiration, return the token. Otherwise, attempt
    * to obtain a new token.
    *
-   * A new token will be obtained either by opening an iframe or a
-   * refresh token (if `useRefreshTokens` is `true`)
+   * A new token will be obtained either by opening an iframe (if `useRefreshTokensFallback` is true) or a
```

Comment:
This addition isn't accurate. Calling `getTokenSilently()` uses iframes by default. You can opt into using refreshtokens by setting `useRefreshTokens`, but `useRefreshTokensFallback` has no implication on that behavior.

When u set `useRefreshTokens` to false, calling `getTokenSilently()` will use an iframe, and the value of `useRefreshTokensFallback` will not even be read.

When u set `useRefreshTokens` to true, calling `getTokenSilently()` will use the refresh tokens, and fallback to iframes when that refreh token call fails.

When we talk about the refresh token fallback, we refer to the iframe approach that is done when the refresh token call failed (not the one that we do when we said not to use refresh tokens).

Reply:
Removed the incorrect claim here